src/components/maskedInput/index.js

```diff
@@ -10,7 +10,7 @@ import Text from '../text';
 /**
  * @description: Mask Input to create custom looking inputs with custom formats
  * @extends: TextInput
- * @extendslink: docs/TagsInput
+ * @extendslink: docs/ChipsInput
```

Comment:
This is actually wrong. 
maskedInput doesn't extend TagsInput
I assume it's an old copy-paste 
you can remove the `extendslink` entry

Reply:
It does extend `TextInput`, so I updated it correctly.